config_src/drivers/nuopc_cap/mom_cap_methods.F90

```diff
@@ -352,14 +354,15 @@ subroutine mom_export(ocean_public, ocean_grid, ocean_state, exportState, clock,
   real                            :: slope, u_min, u_max
   integer                         :: day, secs
   type(ESMF_TimeInterval)         :: timeStep
-  integer                         :: dt_int
+  integer                         :: dt_int,ii,jj
```

Comment:
need to add space between them

Reply:
They were actually unused, so I removed them.